### DISCUSSION TITLE
每次遇到年轻人我就试图给他们洗脑

### DISCUSSION THREAD

**Discussion by surifans** [The-Run-Philosophy-Organization/run#401] [General]
每次遇到年轻人我就试图给他们洗脑：你们要是真有雄心壮志，有能力没背景，我劝你早点把去发达国家和地区作为目标。见了太多因为落后法治不健全，因为一条新规新法整个行业完犊子的例子。又没能力规范好整个行业，只能一刀切！我是个没有本事的人，我从我的祖辈，从我的民族文化中遗传了混吃等死，好死不如赖活着的基因，你们年轻人别这样！

**Comment by letsgow** [comment]
你混吃等死别怪别人好不好，祖宗都不要的人能不能qusi啊。
